### PR TITLE
umi_tools:  clean dependencies

### PR DESCRIPTION
**Pull request by croth1**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [x] This PR does something else (explain below).

removed dependency on certifi. Not sure why it was needed in the first place - also the hard pinning prevents updating a lot of stuff.

### CODE REVIEW

File: recipes/umi_tools/meta.yaml [hunk 1]
```diff
@@ -12,17 +12,18 @@ source:
 
 build:
   number: 1
```
Comment (bgruening) [inline recipes/umi_tools/meta.yaml]: you need to bump the build number here

Comment (croth1) [reply recipes/umi_tools/meta.yaml]: good catch, thanks!

**Comment by croth1** [comment]
Fixes https://github.com/bioconda/bioconda-recipes/issues/11303.

@IanSudbery, if you have some spare time, could you please review the dependencies in the recipe? I am a bit surprised about `freetype` and `pysam`. Do you know whether these are indeed required?

**Comment by IanSudbery** [comment]
Thanks for the fast turnaround! I believe that `freetype` is a dependency of `matplotlib`, but we certainly don't use it directly, and its not on our dependency list. We used to depend on `cython` because we cythonized and complied our extension on the fly with `pyximport`, but we've long since switched to provided the cythonized c file and compiling on install, so it *should* work without it.  I'm happy to run any new recipe through our testing boxes if you tell me how to use a recipe not merged into master.

**Comment by croth1** [comment]
@bgruening, is everything fine with the checks? Why does it not show the CI?

**Comment by croth1** [comment]
@bioconda/core any idea why the CI checks seem to be broken here?

**Comment by rvalieris** [comment]
try merging changes from master perhaps ?

**Comment by croth1** [comment]
@rvalieris is that required? CI ran for the first commit without problems, but now it's stuck.

**Comment by rvalieris** [comment]
its just a guess really, I don't know why its happening either.

**Pull request closed without merging**